homeassistant/components/somfy/cover.py

```diff
@@ -24,8 +24,13 @@ def get_covers():
 
         devices = hass.data[DOMAIN][DEVICES]
 
+        if CONF_SHADOW in hass.data[DOMAIN].keys():
+            shadow = hass.data[DOMAIN][CONF_SHADOW]
+        else:
+            shadow = False
```

Comment:
Useless test as CONF_SHADOW is already fill in the __init__

Reply:
OK, I'll make this change at the weekend.

Reply:
Change made